geoip.go

```diff
@@ -146,6 +146,14 @@ func (gi *GeoIP) GetOrg(ip string) string {
 	return name
 }
 
+// Takes an IPv6 address string and returns the organization name for that IP.
+// Requires the GeoIP organization database.
+func (gi *GeoIP) GetOrg_v6(ip string) string {
```

Comment:
GetOrgV6() to be consistent?

Reply:
I think, your choice is better then my choice about function name.